CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ set(PYBIND11_HEADERS
   include/pybind11/descr.h
   include/pybind11/options.h
   include/pybind11/eigen.h
+  include/pybind11/embedded.h
```

Comment:
I actually like `embed.h` slightly more, given that it's a front-facing user feature, and that, if I'm a user wanting to "embed the interpreter" I think "embed" is the more descriptive word.